monochart/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.20
+version: 1.1.21
```

Comment:
Good as of right now, just need to see which one of ours gets merged first. I'll let you know if mine does.

Reply:
Okay, mine did get merged, so you need to update and bump to 1.2.22. Thanks.

Reply:
```suggestion
version: 1.1.22
```